proxy/envoy/resources.go

```diff
@@ -42,6 +45,9 @@ const (
 	// CDSName is the name of CDS cluster
 	CDSName = "cds"
 
+	// RedirectListenerName is the name for traffic capture listener
+	RedirectListenerName = "redirect"
+
```

Comment:
nit: Lets call it VirtualListenerName .. or MasterListenerName.. Redirect is bit confusing.